scripts/image

```diff
@@ -153,6 +153,9 @@ fi
 # Remote support
   [ "$REMOTE_SUPPORT" = "yes" ] && $SCRIPTS/install remote
 
+# Serdisplib support
+  [ "$SERDISPLIB_SUPPORT" = "yes" ] && $SCRIPTS/install serdisplib
```

Comment:
This isn't needed if you have it listed as a dependency in the lcdproc package already.